src/backend/src/prisma/migrations/20240910005616_add_logo_image_featured_project/migration.sql

```diff
@@ -1,5 +1,5 @@
 -- AlterTable
-ALTER TABLE "Organization" ADD COLUMN     "logoImage" TEXT;
+ALTER TABLE "Organization" ADD COLUMN     "logoImageId" TEXT;
```

Comment:
ensure at some point you consolidate all of your migration files to just one migration called home-page-redesign